asab/log.py

```diff
@@ -295,7 +295,7 @@ class SyslogRFC3164Formatter(StructuredDataFormatter):
 	def __init__(self, fmt=None, datefmt=None, style='%', sd_id='sd'):
 		fmt = '<%(priority)s>%(asctime)s {app_name} {proc_id} %(levelname)s %(name)s %(struct_data)s%(message)s\000'.format(
 			app_name=Config["logging"]["app_name"],
-			hostname=socket.gethostname(),
```

Comment:
What is this?

Reply:
Well, hostname is not used in the string, flake8 was complaining about it.